### DISCUSSION TITLE
`agent-merging`: Add fuzzy logic to recommend duplicates to merge

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5727] [Feature Requests]
In the same vein as https://github.com/specify/specify7/issues/893

**Comment by grantfitzsimmons** [comment]
Potentially using the formatted string to find duplicates

**Comment by maxpatiiuk** [comment]
Turns out Specify 5 had this :sigh:

**Comment by grantfitzsimmons** [comment]
See https://github.com/specify/specify7/issues/893
